kokoro/scripts/windows/build.bat

```diff
@@ -18,8 +18,8 @@ set BUILD_ROOT=%cd%
 set SRC=%cd%\github\amber
 set BUILD_TYPE=%1
 
-choco install cmake --pre --yes --no-progress
-choco upgrade cmake --pre --yes --no-progress
+choco install cmake --pre --yes --no-progress --version 3.15.4
```

Comment:
Why is this needed? This has been working on the bots for the last few weeks?

Reply:
[Here is one failure](https://source.cloud.google.com/results/invocations/ed781401-e336-43ef-8214-088a4def18ff/targets/graphics_shader_compiler%2Famber%2Fwindows-msvc-2017-debug%2Fpresubmit/log)

I am guessing a new pre-release version of CMake was made available that fails to download. But it might not be this. I suggested specifying the latest stable release. I also suggested removing the `--pre` flag.

Reply:
If we drop the --pre, can we drop the --version 3.15.4? That way we don't pin to a specific version we just get the latest released version?

Reply:
Let's try that. Can someone add the tag?